manifests/backup.pp

```diff
@@ -21,6 +26,7 @@
   String  $psql_version             = $pe_databases::psql_version,
```

Comment:
This is failing in CI. The fact in the init.pp is a float instead of a string. We can cast it as a string or handle it as a float here.

Reply:
commit submitted